run-tests.sh

```diff
@@ -52,7 +52,7 @@ if [[ ${keep_services} -eq 0 ]]; then
 fi
 
 python -m check_manifest
-python -m setup extract_messages --dry-run
+python -m setup extract_messages --output-file /dev/null
```

Comment:
Can we avoid hard-coding an absolute path? What was the issue with `--dry-run`?

Reply:
the Problem with `--dry-run` is that, on the local machine the *.mo or *.po files are regenerated. which could lead to a unwanted commit. the /dev/null should be there on unix like system, but i am not sure on a windows OS. so then we would have to remove that and use the approach from mojib